_get_started/previous-versions.md

```diff
@@ -17,6 +17,53 @@ your convenience.
 
 ## Commands for Versions >= 1.0.0
 
+### v1.12.0
+
+#### Conda
+
+##### OSX
+
+```
+# conda
+conda install pytorch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0 -c pytorch
+```
+
+#####  Linux and Windows
+
+```
+# CUDA 10.2
+conda install pytorch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0 cudatoolkit=10.2 -c pytorch
+# CUDA 11.3
+conda install pytorch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0 cudatoolkit=11.3 -c pytorch -c conda-forge
+# CPU Only
+conda install pytorch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0 cpuonly -c pytorch
+```
+
+#### Wheel
+
+##### OSX
+
+```
+pip install torch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0
+```
+
+##### Linux and Windows
+
+```
+# ROCM 4.2 (Linux only)
+pip install torch==1.12.0+rocm4.2 torchvision==0.13.0+rocm4.2 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
+# ROCM 4.1 (Linux only)
+pip install torch==1.12.0+rocm4.1 torchvision==0.13.0+rocm4.1 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
+# ROCM 4.0.1 (Linux only)
+pip install torch==1.12.0+rocm4.0.1 torchvision==0.13.0+rocm4.0.1 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
+# CUDA 11.1
```

Comment:
Is this supported with CUDA 11.1 or 11.3?

Reply:
Yes true cu111 is dropped since 1.11.0 I believe